docs/tutorials/blogs.md

```diff
@@ -0,0 +1,1087 @@
+    indicates that it is a draft. When you run `mkdocs build`, the draft will
+    _not_ appear in the output:
+
+    === "MacOS/Linux"
+        ```
+        $ mkdocs build
+        $ ls site/blog
+        2023		archive		index.html
+        ```
+
+    === "Windows"
+        TODO
+
+    The first blog post for 2024 is not there yet because it is still in draft
+    stage. Remember to remove the `draft` setting in the header when it is time
+    to publish it.
+
+If you are using the [Insiders edition], you can also create
+a folder to keep your drafts in and use the [Meta plugin] to add the
+`draft` header setting to all the posts in that folder. We will cover the
+Meta plugin later on.
+
+[Meta plugin]: ../plugins/meta.md
+
+### Edits
+
+Sometimes, bloggers need to update a post. This might happen when they have made
+a mistake or when something changes they need to reflect in the post. To
+indicate you have edited a post, you can include an edit date in the page header:
+
+!!! example "Editing a post"
+
+    Make a change to your first blog post, then add an edit date to the header:
+
+    ```hl_lines="3"
+    ---
+    date: 2023-12-31
+    updated: 2024-01-02
+    ---
+    ```
+
+### Reading time
+
+To give the reader some idea of how long it might take them to read a post,
+you can specify a reading time. Of course, reading times vary by individual as
+well as by the subject matter and writing style. So, how do you set a reading
+time for your content? Absent any other data, it seems sensible to start with
+a reading speed used by big blogging platforms and adjust downward somewhat
+if the content is fairly technical. [Medium use] a reading speed of 265 words
+per minute. So, count the number of words in a post and divide by your reading
+speed value to get the time, then round to the nearest minute. For content
+in Chinese, Korean, and Japanese, use the value of 500 characters per minute
+instead of words per minute.
+
+[Medium use]:https://help.medium.com/hc/en-us/articles/214991667-Read-time
+
+!!! example "Setting reading time"
+
+    Add a reading time
+
+    ```
+    ---
+    date: 2023-12-31
+    updated: 2024-01-02
+    readtime: 15
+    ---
+    ```
+
+### Pinning
+
+Sometimes, blog authors want to 'pin' a specific post so that it will always
+appear at the top of the index page, no matter what else gets published. If you
+are using the [Insiders edition], you can achieve this by adding the `pin`
+attribute in the page header:
+
+!!! example "Pin a post <!-- md:sponsors -->"
+
+    Add the `pin` attribute to your first blog post:
+
+    ```hl_lines="5"
+    ---
+    date: 2023-12-31
+    updated: 2024-01-02
+    readtime: 15
+    pin: true
+    ---
+    ```
+
+    Observe how this makes the post appear on top of the index page even though
+    its publication date is prior to other posts.
+
+### Custom slugs
+
+Another useful header attribute is `slug`, which allows you to define the slug
```

Comment:
There's not mention whatsoever about the [configuration options that influence slug creation](https://squidfunk.github.io/mkdocs-material/plugins/blog/#config.post_slugify). Manually defining slugs should be a last resort.

Reply:
I'd suggest removing this, then. Pointing to configuration options involving Python code will not sit well in an introductory tutorial. Let's talk about how to split the tutorial up and what to include or not in the first one.